hoomd/SnapshotSystemData.h

```diff
@@ -54,10 +54,7 @@ template<class Real> struct SnapshotSystemData
     PairData::Snapshot pair_data;             //!< The pair data
 
     //! Constructor
-    SnapshotSystemData()
-        {
-        dimensions = 3;
-        }
+    SnapshotSystemData() : dimensions {3}, global_box {new BoxDim()} { }
```

Comment:
```suggestion
    SnapshotSystemData() : dimensions(3), global_box(new BoxDim()) { }
```
Use an initialization style consistent with the rest of the code base.